traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -236,11 +236,63 @@ <h4>New Term Information</h4>
       </div>
     </section>
     {% endif %}
+
+    <section class="section">
+      <h4>TRAIT HISTORY</h4>
```

Comment:
Two comments about the trait history header section:
![image](https://user-images.githubusercontent.com/10669118/89111342-96c02800-d44c-11ea-83e2-050f2d832a80.png)

1. For individual comments the left border looks good as an UI element. However, I think that for the “Comment as” section we should get rid of it, and align it on the same level with the "TRAIT HISTORY" title, in order to distinguish it from the actual comments and historical actions below.
2. As you can see, for some reason the input field is rendered in such a way that it has a tiny scrollbar. The styles probably need to be tweaked a bit to get rid of it. This might be something specific to Chrome.

Reply:
That is indeed a weird Chrome specific issue and it looks like quite a pain because if I just disable the scrollbar, then the user wouldn't be able to scroll up an down if the comment size exceeds the text area height. So I think the best approach here would be to hide the scrollbar when the text area is collapsed, and show it when it is expanded.

Reply:
Yes, this approach works great, I just tested it. I'll resolve this conversation then, great job